archiva-modules/archiva-web/archiva-rest/archiva-rest-services/src/main/java/org/apache/archiva/rest/services/DefaultRepositoriesService.java

```diff
@@ -423,7 +423,7 @@ public Boolean copyArtifact( ArtifactTransferRequest artifactTransferRequest )
             String timestamp = null;
 
             File versionMetadataFile = new File( targetPath, MetadataTools.MAVEN_METADATA );
-            ArchivaRepositoryMetadata versionMetadata = getMetadata( versionMetadataFile );
+            /* unused */ getMetadata( versionMetadataFile );
```

Comment:
Probably can be removed as the metadata is not used. However since the method throws I kept it. Can if this is used to verify that meta data exists it should be probably documented.